nbproject/project.properties

```diff
@@ -0,0 +1,3 @@
+files.encoding=UTF-8
```

Comment:
Do we want to check in anything in this directory?  You added '/nbproject/private/' in the .gitignore, maybe you want '/nbproject/'.